### PR TITLE
app-admin/aerospike-amc-community: various changes (read first comment)

### PR DESCRIPTION
**Pull request by scardracs**
Bump to new version
Removed inherits
Fixed GLEP 81
Added acct-group/user

Package-Manager: Portage-3.0.18, Repoman-3.0.3
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

**Comment by scardracs** [comment]
@thesamesam @juippis be sure to check acct-user and group as I have set -1 as value :)

File: app-admin/aerospike-amc-community/aerospike-amc-community-4.1.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Web UI based monitoring tool for Aerospike Community Edition Server"
+HOMEPAGE="https://www.aerospike.com"
+SRC_URI="https://github.com/aerospike-community/amc/releases/download/4.1.3/aerospike-amc-enterprise-4.1.3-linux.tar.gz -> ${P}.tar.gz"
```
Comment (telans) [inline app-admin/aerospike-amc-community/aerospike-amc-community-4.1.3.ebuild:8]:
```suggestion
SRC_URI="https://github.com/aerospike-community/amc/releases/download/${PV}/aerospike-amc-enterprise-${PV}-linux.tar.gz -> ${P}.tar.gz"
```

File: app-admin/aerospike-amc-community/files/aerospike-amc-community-4.0.19-r1.ebuild [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
 
-S=${WORKDIR}
+S="${WORKDIR}"
 
 src_install() {
-	cp -r "$S/"* "$D"
-	rm "${D}/etc/init.d/*"
-	newinitd "${FILESDIR}/amc.init.4" amc
+	cp -r "${S}/"* "${D}"
+	rm "${D}"/etc/init.d/*
```
Comment (telans) [inline app-admin/aerospike-amc-community/files/aerospike-amc-community-4.0.19-r1.ebuild:25]:
```suggestion
	cp -r "${S}/"* "${ED}" || die
	rm "${ED}"/etc/init.d/* || die
```

File: app-admin/aerospike-amc-community/aerospike-amc-community-4.1.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	acct-user/aerospike-amc-community
+"
+
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}"
+
+src_install() {
+	cp -r "${S}"/* "${D}"
+	rm "${D}"/etc/init.d/amc || die
```
Comment (telans) [inline app-admin/aerospike-amc-community/aerospike-amc-community-4.1.3.ebuild:25]:
```suggestion
	cp -r "${S}"/* "${ED}" || die
	rm "${ED}"/etc/init.d/amc || die
```

**Pull request closed without merging**